documentation/docs/getting_started/installation/nodejs/local_setup.md

```diff
@@ -0,0 +1,88 @@
+---
+image: /img/integration-services/logo/integration_services.png
+description: This section will guide in setting up the Integration Service API using NodeJs.
+keywords:
+- nodejs
+- setup API
+- configure
+- .env
+---
+
+# Local Setup
+
+This section will guide in setting up the Integration Service API using NodeJs.
+
+## Requirements
+
+Please make sure to have the following installed before moving forward:
+
+* [node.js](https://nodejs.org/)
+
+## Download the Project
+
+1. Clone the project by running the following command:
+
+    ```bash
+    git clone https://github.com/iotaledger/integration-services.git
+    ```
+
+2. Change directory into the `api` folder in your freshly cloned project by running:
+
+    ```bash
+    cd integration-services/api
+    ```
+
+## Configuration
+
+If this is your first time installing the project, please make a copy of
+the [.env.example file](https://github.com/iotaledger/integration-services/blob/master/api/.env.example) and rename it
+as `.env`, and also copy
```

Comment:
Just in case, I think some text may be missing here.  Looking at the next paragraph I think it may be https://github.com/iotaledger/integration-services/blob/master/api/mongo-init.js.example 
```suggestion
as `.env`, and also copy the [mongo-init.js.example](https://github.com/iotaledger/integration-services/blob/master/api/mongo-init.js.example) and rename it as `mongo.init.js`.
```